src/Workspaces/CSharp/Portable/Simplification/CSharpSimplificationService.cs

```diff
@@ -90,6 +91,11 @@ public static SyntaxToken TryEscapeIdentifierToken(SyntaxToken syntaxToken, Synt
                 return syntaxToken;
             }
 
+            if (SyntaxFacts.GetContextualKeywordKind(syntaxToken.ValueText) == SyntaxKind.UnderscoreToken)
+            {
+                return syntaxToken;
+            }
```

Comment:
I think this is the right place. `_` and `@_` may not have the same meaning, and there are multiple callers of `TryEscapeIdentifierToken` which may benefit from this common knowledge/behavior.

Let me know.